### PR TITLE
PCHR-1720: Adds Styles form PCHR-1353 + fix for labels

### PR DESCRIPTION
**Pull request by bmnepali**
## Issue:
Styles form PCHR-1353 were not merged which halted css preprocessing (SASS). Labels for the Contact->Personal Details ->Extended Demographics edit form were hidden, as a result they were invisible. 

## Solution:
1. Styles form PCHR-1353 were extracted and applied in this fix.
2. Added style for labels for Extended Demographics to be visible.

### Before:
extdem

### After:
screen shot 2017-01-05 at 3 15 27 pm

### CODE REVIEW

File: org.civicrm.bootstrap/scss/overrides/reset/_drupal.scss [hunk 1]
```diff
@@ -1,672 +1,46 @@
...
+ *
+ * (`initial` was used at first, but there is no support by IE for it)
+ */
+h3 {
+  @include initial(background-color, transparent);
+  @include initial(font-family, $font-family-base);
+  @include initial(padding, 0);
+  @include initial(margin, 0);
 
-// Media query variables.
```
Comment (AkA84) [inline org.civicrm.bootstrap/scss/overrides/reset/_drupal.scss]: Why does this file has so many lines removed?

File: org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php [hunk 2]
```diff
@@ -121,18 +109,16 @@ function bootstrapcivicrm_civicrm_angularModules(&$angularModules) {
...
   _bootstrapcivicrm_civix_civicrm_alterSettingsFolders($metaDataFolders);
 }
-
 /**
  * Implementation of hook_civicrm_pageRun
  */
 function bootstrapcivicrm_civicrm_pageRun($page) {
   if (!(isset($_GET['snippet']) && $_GET['snippet'] == 'json')) {
-    CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css');
+    CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivihr', 'css/civihr.css');
```
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php]: Why are you referencing a completely different extension and file?

Comment (bmnepali) [inline org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php]: This is extracted form PCHR-1353 branch.

File: org.civicrm.bootstrap/scss/mixins/_body-namespaced.scss [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
  */
-@mixin body-namespaced {
-    @at-root {
-        body#{$bootstrap-namespace},
-        #{$bootstrap-namespace} {
-            @content;
-        }
-    }
-}
+// @mixin body-namespaced {
```
Comment (AkA84) [inline org.civicrm.bootstrap/scss/mixins/_body-namespaced.scss]: You removed this partial before, now you are commenting it out. Why?

File: org.civicrm.bootstrap/scss/overrides/namespace/_scaffolding.scss [hunk 4]
```diff
@@ -1,164 +1,164 @@
+// //
+// // Scaffolding for namespaced theme
```
Comment (AkA84) [inline org.civicrm.bootstrap/scss/overrides/namespace/_scaffolding.scss]: You removed this partial before, now you are commenting it out. Why?

File: org.civicrm.bootstrap/scss/overrides/reset/_drupal.scss (hunk 1 above)
Comment (AkA84) [inline org.civicrm.bootstrap/scss/overrides/reset/_drupal.scss]: Can you answer this please?

**Pull request merged**